docs/core/stories.rst

```diff
@@ -51,7 +51,46 @@ This is what we call a **story**.
   and contain the name of the action.
 - Events returned by an action are on lines immediately after that
   action. For example, if an action returns a ``SetSlot`` event,
-  this is shown as the line ``- slot{"slot_name": "value"}``
+  this is shown as the line ``- slot{"slot_name": "value"}``.
+
+Let's now take a slightly more detailed look at each of these components,
+along with some things you should keep in mind while using them to write
+your own stories.
+
+User Messages
+~~~~~~~~~~~~~
+While writing stories, you do not have to deal with the specific contents of
+the messages that the users send. Instead, you can take advantage of the output
+from the NLU pipeline, which lets you use just the combination of an intent and
+entities to refer to all the possible messages the users can send to mean the
+same thing.
+
+It is important to include the entities as well because the policies learn to
+predict the next action based on a *combination* of both the intent and
+entities.
+
+Actions
+~~~~~~~
+While writing stories, you will encounter two types of actions - utterances
+and custom actions. Utterances are hardcoded message that a bot can respond
+with. Custom actions, on the other hand, involve custom code being executed. 
+
+All actions (both utterances and custom actions) executed by the bot are shown
+as lines starting with ``-`` followed by the name of the action. For custom
+actions, the action name is the string you choose to return from the ``name``
+method of the custom action class. The convention is that utterances begin
+with the prefix ``utter_`` and custom actions begin with ``action_``.
+
+Events
+~~~~~~
+Events such as setting a slot or activating/deactivating a form have to be
```

Comment:
Forms and slots are the **only** events that are included in story data.  So I would write a subsection for each instead of this, so that you can drill home the message that slot setting steps are important

Reply:
I agree with your point that some amount of description for slot and form events is required here, but that would result in redundant information existing in both the stories page and the forms page. What about simply linking them to the respective pages?